### ISSUE TITLE
Does not support hash routes

### ISSUE THREAD

**Issue by kamranayub** [scottrippey/next-router-mock#29]
In next/router proper, I am seeing `asPath` values that include the URL hash as I expect. However, passing hashes to `next-router-mock` is removing hashes. It looks like the pathing logic only deals with `pathname` and `query` but doesn't preserve `hash` so my tests are "passing" even though at runtime the behavior is different (and failing) causing a false-positive test suite.

```js
router.push('/page#hash=true');
router.push(new URL('/page#hash=true', 'https://www.google.com'));

console.log(asPath);
// logs: "/page"
```

**Comment by scottrippey** [comment]
Fixed in `0.6.3`!

**Comment by kamranayub** [comment]
You the best!
